pkgs/by-name/an/anubis/package.nix

```diff
@@ -32,6 +66,14 @@ buildGoModule (finalAttrs: {
       "-extldflags=-static"
     ];
 
+  postPatch = ''
+    patchShebangs ./web/build.sh
+  '';
+
+  preBuild = ''
+    go generate ./... && ./web/build.sh && cp -r ${anubisXess}/xess.min.css ./xess
```

Comment:
FYI: we usually write one command per line and don't concatenate them with && for improved readability.

Reply:
Ah, didn't know. I'll remember for next time